multimodal/src/autogluon/multimodal/models/clip.py

```diff
@@ -169,6 +169,7 @@ def forward(
                 return_dict=True,
             )
             text_features = self.model.text_projection(text_outputs.pooler_output)  # (b, num_features)
+            last_hidden_state = text_outputs[0]
```

Comment:
Does `text_outputs.last_hidden_state` not work for TensorRT?

Reply:
This has to be changed, since we are adding the `torchscript=True` argument in 
```
AutoConfig.from_pretrained(checkpoint_name, torchscript=True)
```

Basically, `text_outputs[0]` (with `torchscript=True`) equals to `text_ouputs.last_hidden_state` (with `torchscript=False`).

Reply:
Since we don't use `torchscript=True`, do we still need `last_hidden_state = text_outputs[0]`?

Reply:
undid this changed.